### ISSUE TITLE
[BUG]: Help loading is slow

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#1318]
### Project

a11y checker extension

### Browser

Chrome, Firefox

### Operating system

MacOS, Windows

### Description

From external contact:
> I really enjoy using the Equal Access Accessibility Checker, but after scanning a site, when I click on the issue the right window won't load or takes over 45 secs to load. Switching to another violation can also take the explanation a long time to load. I found the same thing happens in Chrome and Firefox, and on a PC and a Mac.

I have also seen this behavior occasionally. We're using the unpkg open source CDN to load the help. Once a particular file is accessed, reloading it is fairly fast, but the initial load is very slow because it's not in their cache yet. Unfortunately, JSDelivr does not support viewing HTML directly. We may need to change back to hosting the help files ourselves until we find a better CDN.

### Steps to reproduce

Scan and flip through the help.

**Comment by aliunwala** [comment]
Triage:
We might need to update links in several places (Excel sheet and extension). But it is "felt" most on the extension and we should fix it there first. 

Another possible solution is to hit the Unpkg CDN from time to time/or when a scan is made to refresh the files that are recently used in the cache. (Might run into sometype of limit here from unpkg)

**Comment by tombrunet** [comment]
Trying something different in branch `2023-ext-arch`. It will pull from the unpkg CDN and our own servers and fastest one wins. Also changed `latest` to resolve the version locally so that unpkg doesn't have to fetch, redirect, and fetch again. So, this is potentially fixed, but won't see it until we get the `2023-ext-arch` work is done.
